pkg/model/datacatalog/api.go

```diff
@@ -21,6 +21,8 @@ type GetAssetResponse struct {
 	// Vault plugin path where the data credentials will be stored as kubernetes secrets
 	// This value is assumed to be known to the catalog connector.
 	Credentials string `json:"credentials"`
+	// Additional message to be reported to the user
```

Comment:
Should we provide more info about the message format (the code, level, ....)

Reply:
No, the message format is specific to the catalog in use.